<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>COnlineList</class>
 <widget class="QWidget" name="COnlineList">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QWidget" name="widgetList" native="true">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QTableWidget" name="tableWidget">
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::SingleSelection</enum>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
        <property name="rowCount">
         <number>50</number>
        </property>
        <property name="columnCount">
         <number>6</number>
        </property>
        <attribute name="horizontalHeaderCascadingSectionResizes">
         <bool>true</bool>
        </attribute>
        <attribute name="horizontalHeaderHighlightSections">
         <bool>false</bool>
        </attribute>
        <attribute name="horizontalHeaderStretchLastSection">
         <bool>true</bool>
        </attribute>
        <attribute name="verticalHeaderVisible">
         <bool>false</bool>
        </attribute>
        <attribute name="verticalHeaderHighlightSections">
         <bool>false</bool>
        </attribute>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <column>
         <property name="text">
          <string>ID号</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>软件地址</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>出厂编号</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>硬件地址</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>上线时间</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>软件版本</string>
         </property>
        </column>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget" native="true">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>200</height>
         </size>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="2" column="0">
          <widget class="QLabel" name="label">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>35</height>
            </size>
           </property>
           <property name="text">
            <string>发送文件</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_3">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>35</height>
            </size>
           </property>
           <property name="text">
            <string>发送命令</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="lineEditCmd">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>35</height>
            </size>
           </property>
           <property name="text">
            <string>1007</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="lineEditFile">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>35</height>
            </size>
           </property>
           <property name="text">
            <string>/mnt/nfs/001.cpp</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_2">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>35</height>
            </size>
           </property>
           <property name="text">
            <string>接收数据</string>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QPushButton" name="btnCmd">
           <property name="minimumSize">
            <size>
             <width>97</width>
             <height>44</height>
            </size>
           </property>
           <property name="text">
            <string>发送命令</string>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QPushButton" name="btnFile">
           <property name="minimumSize">
            <size>
             <width>97</width>
             <height>44</height>
            </size>
           </property>
           <property name="text">
            <string>发送文件</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="lineEdit">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>35</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widgetBtn" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="btnExit">
        <property name="minimumSize">
         <size>
          <width>97</width>
          <height>44</height>
         </size>
        </property>
        <property name="text">
         <string>返回</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnLoad">
        <property name="minimumSize">
         <size>
          <width>97</width>
          <height>44</height>
         </size>
        </property>
        <property name="text">
         <string>刷新列表</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnRead">
        <property name="minimumSize">
         <size>
          <width>60</width>
          <height>44</height>
         </size>
        </property>
        <property name="text">
         <string>读取配置</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnTest">
        <property name="minimumSize">
         <size>
          <width>60</width>
          <height>44</height>
         </size>
        </property>
        <property name="text">
         <string>进入测试</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
